openpower/package/op-test/op-test.mk

```diff
@@ -0,0 +1,23 @@
+################################################################################
+#
+# op-test
+#
+################################################################################
+
+OP_TEST_VERSION = $(call qstrip,$(BR2_OP_TEST_VERSION))
+OP_TEST_SITE = $(call github,open-power,op-test,$(OP_TEST_VERSION))
+
+OP_TEST_LICENSE = Apache-2.0
+
+OP_TEST_LICENSE_FILES = LICENSE
+
+OP_TEST_INSTALL_TARGET = NO
+OP_TEST_INSTALL_IMAGES = YES
+
+# surely there's an easier way to make a "clean" copy of the git tree
+define OP_TEST_INSTALL_IMAGES_CMDS
+	mkdir -p $(BINARIES_DIR)/op-test
+	tar -C $(BINARIES_DIR)/op-test/ -xf $(OP_TEST_DL_DIR)/op-test-$(OP_TEST_VERSION).tar.gz
```

Comment:
This might work (untested):

```
git archive  --prefix=op-test -o op-test-$(OP_TEST_VERSION).tar.gz $(OP_TEST_VERSION)
```

Reply:
Doesn't work since it the downloaded tarball is just a snapshot rather than a repo clone.